src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -4,12 +4,26 @@ import _ from 'lodash';
 
 // TO DO: Replace made up properties and content with real versions once finalized.
 export const STATUS_TYPES = {
-  nod: 'nod',
-  pendingForm9: 'pending_form9',
-  awaitingHearingDate: 'awaiting_hearing_date',
+  scheduledHearing: 'scheduled_hearing',
+  pendingHearingScheduling: 'pending_hearing_scheduling',
   onDocket: 'on_docket',
+  pendingCertificationSsoc: 'pending_certification_ssoc',
+  pendingCertification: 'pending_certification',
+  pendingForm9: 'pending_form9',
+  pendingSoc: 'pending_soc',
+  stayed: 'stayed',
+  atVso: 'at_vso',
+  bvaDevelopment: 'bva_development',
+  decisionInProgress: 'decision_in_progress',
   bvaDecision: 'bva_decision',
-  remand: 'remand'
+  fieldGrant: 'field_grant',
+  withdrawn: 'withdrawn',
+  ftr: 'ftr',
+  ramp: 'ramp',
```

Comment:
Should this still be here? ([Source](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7937#issuecomment-358378694))